packages/medusa/src/services/claim-item.ts

```diff
@@ -16,45 +35,22 @@ class ClaimItemService extends BaseService {
     lineItemService,
     eventBusService,
   }) {
-    super()
+    // eslint-disable-next-line prefer-rest-params
+    super(arguments[0])
```

Comment:
suggestion (non-blocking): maybe it's time to disable this rule from `.eslintrc.js` in the root of the monorepo?